tests/Unit/Infrastructure/Processing/AssetProcessingJobPayloadTest.php

```
@@ -0,0 +1,59 @@
+final class AssetProcessingJobPayloadTest extends TestCase
+{
+    private const ASSET_ID = '123e4567-e89b-42d3-a456-426614174000';
+
+    #[Test]
+    public function itRoundTripsSerializedPayloads(): void
+    {
+        // Arrange
+        $payload = new AssetProcessingJobPayload(self::ASSET_ID, 3);
+
+        // Act
+        $decodedPayload = AssetProcessingJobPayload::fromJson($payload->toJson());
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Infrastructure\Processing\AssetProcessingJobPayload' in method 'itRoundTripsSerializedPayloads'. (undefined)

(StaticAccess)